### ISSUE TITLE
Multitouch not working in 3.4.3 Release in iOS

### ISSUE THREAD

**Issue by SkanerSoft** [godotengine/godot#58550]
### Godot version

Godot 3.4.3 Release

### System information

iOS 15.2.1 / iPhone SE / OpenGL ES 2.0 Renderer: Apple A9 GPU

### Issue description

First there were big glitches with touches in 3.4.3 RC 2 and I couldn't test multi-touches in it because of this issue - #58274

In the new version, this problem was fixed, but multitouch does not work completely.

### Steps to reproduce

The problem concerns both the _input and the SreenTouchButton events.

I have no way to run deeper tests to find out the cause of the problem, but this again prevents me from releasing the game for iOS...

### Minimal reproduction project

Any project with touch input control.

**Comment by naithar** [comment]
On my own project I wasn't able to reproduce an issue. I guess an MRP is required for this one.

**Comment by SkanerSoft** [comment]
> @SkanerSoft please try this branch [naithar@f6fbe06](https://github.com/naithar/godot/commit/f6fbe069c6ec98b3c5513e70e5d3578a9aa692c1) you should be able to use GitHub built template from here: https://github.com/naithar/godot/actions/runs/1904075045 once it's finished.

Unfortunately, I don't have that option. The build of the game is being done by a friend of mine who is currently unavailable for the next few days.

@naithar

**Comment by naithar** [comment]
I'm getting this result with the fix.

https://user-images.githubusercontent.com/3750083/155875598-188bc4e0-3688-4875-977b-7d6109e491d8.MP4

**Comment by SkanerSoft** [comment]
The fix probably works, but I can't test it yet. I don't have a macbook to compile the game and my friend can't do it right now. I believe it works and am waiting for the 3.5 beta 2 update to test it out. Hope it comes out as soon as possible! Thanks for your work!

**Comment by akien-mga** [comment]
@SkanerSoft The fix is included in 3.4.4 RC 1 released today: https://godotengine.org/article/release-candidate-godot-3-4-4-rc-1
It would be great if you could confirm that it works fine for your project before we release 3.4.4 stable.

**Comment by CarlosRios** [comment]
Hey guys, I've been trying to implement multitouch in my game today and came across this thread. I'm building primarily for IOS and I can confirm after installing 3.4.4 RC1 and then building out the game, that multitouch still isn't working for me. Has anyone else tested it on hardware?

I tested on an iPhone 8, which is my primary testing device.

**Comment by naithar** [comment]
Just tested `3.4.4.rc1` from https://downloads.tuxfamily.org/godotengine/3.4.4/rc1/ with this code:
```

func _unhandled_input(event):
	if event is InputEventScreenTouch:
		print("Index: " + str(event.index) + ", pressed: " + str(event.pressed))

```

Had to rename templates folder from `3.4.4-rc1` to `3.4.4.rc1` to force Export to work.

Got this logs from XCode project:

```
2022-03-15 02:45:29.629209+0300 Test Input[11611:2860390] Index: 0, mode: True
Index: 0, mode: True
2022-03-15 02:45:30.024724+0300 Test Input[11611:2860390] Index: 1, mode: True
Index: 1, mode: True
2022-03-15 02:45:31.459144+0300 Test Input[11611:2860390] Index: 2, mode: True
Index: 2, mode: True
2022-03-15 02:45:32.076045+0300 Test Input[11611:2860390] Index: 3, mode: True
Index: 3, mode: True
2022-03-15 02:45:33.809497+0300 Test Input[11611:2860390] Index: 4, mode: True
Index: 4, mode: True
2022-03-15 02:45:34.660112+0300 Test Input[11611:2860390] Index: 2, mode: False
Index: 2, mode: False
2022-03-15 02:45:35.456377+0300 Test Input[11611:2860390] Index: 3, mode: False
Index: 3, mode: False
2022-03-15 02:45:36.707706+0300 Test Input[11611:2860390] Index: 1, mode: False
Index: 1, mode: False
2022-03-15 02:45:37.141145+0300 Test Input[11611:2860390] Index: 0, mode: False
Index: 0, mode: False
2022-03-15 02:45:37.488677+0300 Test Input[11611:2860390] Index: 4, mode: False
Index: 4, mode: False
```

Fix seems to be working.

**Comment by CarlosRios** [comment]
You're right! error was that I hadn't downloaded the new export templates. Changing the folder did it for me as well.

**Comment by akien-mga** [comment]
> Had to rename templates folder from `3.4.4-rc1` to `3.4.4.rc1` to force Export to work.

Oops, I'll have to fix my scripts to prevent this from happening.

**Comment by SkanerSoft** [comment]
> It would be great if you could confirm that it works fine for your project before we release 3.4.4 stable

@akien-mga 

Hey! Yes, it works well. It's true that the octahedral compression issue hasn't gone away on iOS, but that will be in another post. Not here.

**Comment by akien-mga** [comment]
@SkanerSoft There has been more fixes for octahedral compression after 3.4.4-rc1, I will release 3.4.4-rc2 today or tomorrow to validate them.
